packages/material-ui/src/FormControl/index.js

```diff
@@ -1 +1,2 @@
 export { default } from './FormControl';
+export { useFormControl } from './FormControlContext';
```

Comment:
What do you think of merging this hook with the `formControlState` helper? https://github.com/mui-org/material-ui/issues/11865#issuecomment-500482693

Reply:
I'm asking regarding what API we should expose.

Reply:
Just expose the state? Why do you think formControlState should be included?

Reply:
I personally have a slight preference for not exposing `formControlState`. This is a module we use for merging the context with the props values. People asking for accessing the form state might not need it. However, I'm asking because people tracking #11865 might expect `formControlState` to be public based on my initial comment. Should we collect their feedback?

Reply:
It's just merging objects. I ... yeah no idea.

Reply:
It's just merging objects. I ... yeah no idea.